internal/app/registry_service/service/service.go

```diff
@@ -21,31 +21,31 @@ func NewEquipment(repo EquipmentRepo) Equipment {
 	return Equipment{repo: repo}
 }
 
-func (eqs *Equipment) List() ([]model.Equipment, error) {
-	return eqs.repo.List()
+func (service *Equipment) List() ([]*dtos.EquipmentGet, error) {
+	return service.repo.List()
 }
 
-func (eqs *Equipment) Create(eqt model.EquipmentType, eqp model.Params) (uuid.UUID, error) {
-	return eqs.repo.Create(eqt, eqp)
+func (service *Equipment) Create(eqc *dtos.EquipmentCreate) (uuid.UUID, error) {
+	return service.repo.Create(eqc)
 }
 
-func (eqs *Equipment) Update(id uuid.UUID, eqos *model.OperationalStatus, eqp model.Params) error {
-	return eqs.repo.Update(id, eqos, eqp)
+func (service *Equipment) Update(equ *dtos.EquipmentUpdate) (bool, error) {
+	return service.repo.Update(equ)
 }
 
-func (eqs *Equipment) Get(eqId string) (*model.Equipment, error) {
-	id, err := uuid.FromString(eqId)
+func (service *Equipment) Get(ids string) (*dtos.EquipmentGet, error) {
+	id, err := uuid.FromString(ids)
 	if err != nil {
 		return nil, err
```

Comment:
create a new Error
`return false, fmt.Errorf("Get: failed to convert string '%s' to UUID: %w", eqId, err)`

Reply:
Done.